Notify/Notify/Notify/ViewModels/BluetoothSettingsPageViewModel.cs

```diff
@@ -41,6 +48,31 @@ private async void onBackButtonClicked()
         {
             await Shell.Current.GoToAsync(Constants.SHELL_NAVIGATION_SETTINGS);
         }
+        
+        public string SelectedLocation
+        {
+            get => m_SelectedLocation;
+            set
+            {
+                if (SetField(ref m_SelectedLocation, value))
+                {
+                    string destinationsJson = Preferences.Get(Constants.PREFERENCES_DESTINATIONS, string.Empty);
+                    List<Destination> destinations = JsonConvert.DeserializeObject<List<Destination>>(destinationsJson);
+                    bool isDestinationExists = destinations.Any(destination => destination.Name == m_SelectedLocation);
+
+                    if (isDestinationExists)
+                    {
+                        RemoveBluetoothButtonText = $"REMOVE {value} BLUETOOTH";
+                        IsRemoveButtonEnabled = true;
+                    }
+                    else
+                    {
+                        RemoveBluetoothButtonText = $"No {m_SelectedLocation} destination defined";
+                        IsRemoveButtonEnabled = false;
+                    }
```

Comment:
Instead using a new variable called "isDestinationExists", use the "IsRemoveButtonEnabled" member.